net/tstun/wrap.go

```diff
@@ -325,26 +339,21 @@ func (t *Wrapper) IdleDuration() time.Duration {
 }
 
 func (t *Wrapper) Read(buf []byte, offset int) (int, error) {
-	var n int
-
+	res := <-t.outbound
```

Comment:
how does a close unblock this?

Reply:
It didn't, and that mattered. Now it does.